src/sql/create-constraints.sql

```diff
@@ -304,6 +306,61 @@ where
     and transport_flow_info.outgoing_flows_have_transport_flows
     -- Assets with unit commitment already have a minimum outgoing flow constraints
     and not asset.unit_commitment
+    and asset.investment_method in ('compact','simple', 'none')
+;
+
+drop sequence id
+;
+
+create sequence id start 1
+;
+
+drop table if exists cons_min_outgoing_flow_for_transport_vintage_flows
+;
+
+-- This constraint is very similar to cons_min_outgoing_flow_for_transport_flows_without_unit_commitment
+-- but it applies to vintage flows instead of regular flows
+create table cons_min_outgoing_flow_for_transport_vintage_flows as
+with
+    transport_flow_info as (
+        select
+            asset.asset,
+            coalesce(
+                (
+                    select
+                        bool_or(flow.is_transport)
+                    from
+                        flow
+                    where
+                        flow.from_asset = asset.asset
+                ),
+                false
+            ) as outgoing_flows_have_transport_flows,
+        from
+            asset
+    )
+select
```

Comment:
It might be good to benchmark this, just to be sure there are no surprises. Maybe run locally if its easier than the branch change.